tools/ci/generate_sql_scripts.py

```diff
@@ -15,8 +16,12 @@
 orderedSqlFiles = sorted(orderedSqlFiles)
 
 for index in orderedSqlFiles:
-    # Yes I know half these casts are probably not necassary, but python is very picky
-    orderedSqlFiles[index] = str(index) + "-" + (str(int(index)+1)) + ".sql"
+    # Yes I know half of the casts below this are probably not necassary, but python is very picky
+    # AND YES I KNOW THIS IS SNOWFLAKEY AS HELL, BUT IT MUST BE DONE FOR PROPER CI
+    if index in [16, 17]:
+        orderedSqlFiles[index] = str(index) + "-" + (str(int(index)+1)) + ".py"
```

Comment:
Easy dumb way to write this better:
```suggestion
        orderedSqlFiles[index] = "%d-%d.py" % (index, index+1)
```
More fancy way:
```suggestion
        orderedSqlFiles[index] = "{}-{}.py".format(index, index+1)
```
Fanciest way (if you have python 3.6+, which is likely, but idk what your CI uses):
```suggestion
        orderedSqlFiles[index] = f"{index}-{index+1}.py"
```

Same below.

Reply:
The script works fine as it is, and its with code I actually fully understand like the back of my hand (Even though I wrote `generate_sql_scripts.py`, I am not a python wizard). If other maints ask I will change it to be this way, but as of now I would rather keep it in a way I can understand.